src/error/syntaxError.js

```diff
@@ -22,9 +22,11 @@ export function syntaxError(
   description: string
 ): GraphQLError {
   const location = getLocation(source, position);
+  const line = location.line + source.location.line - 1;
```

Comment:
How about `locationOffset`?

Oh right! I meant to mention that in the pull request body. I wrote code to adjust for the column offset, but deleted it because it didn't seem worth the hassle for a pretty rare (as far as I can tell) special case. I can add the code again if you think it's important.